src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/Renamer.HandleFileMove.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Rename;
+using Microsoft.VisualStudio.ProjectSystem.Waiting;
+using Solution = Microsoft.CodeAnalysis.Solution;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Rename
+{
+    internal partial class RenamerProjectTreeActionHandler
```

Comment:
>RenamerProjectTreeActionHandler [](start = 27, length = 31)

Based on this class name, I think the convention would be for this file to be called RenamerProjectTreeActionHandler.HandleFileMove (the prefix matching the class name)

Reply:
I agree with you, but Undo is also part of this partial class and I just followed that same convention.
Maybe we should rename both to follow RenamerProjectTreeActionHandler.HandlerFileMove

![image](https://user-images.githubusercontent.com/510598/94070681-7c992b00-fda7-11ea-999d-6be6b170982a.png)

Reply:
Good point, existing convention is file to use

---
In reply to: [493901852](https://github.com/dotnet/project-system/pull/6628#discussion_r493901852) [](ancestors = 493901852)